src/apps/index/styles/nav.sass

```diff
@@ -0,0 +1,87 @@
+@import './variables.sass'
+$nav-width: 450px
+
+body
+  margin-left: $nav-width
+
+  @media (max-width: $mobile-max-width)
+    margin-left: 0
+
+nav
+  background-image: url('https://images.unsplash.com/photo-1502485582557-aee19598572c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1952&q=80')
+  background-size: cover
+  background-position: center
+  height: 100vh
+  width: $nav-width
+  padding: 4rem
+  display: flex
+  flex-direction: column
+  align-items: center
+  justify-content: flex-start
+  font-family: Lobster, cursive, serif
+  position: fixed
+  left: 0
+  top: 0
+
+  @media (max-width: $mobile-max-width)
+    position: relative
+    width: 100%
+    height: 100vh
+    justify-content: space-between
+
+  .profile
+    width: 100%
+    height: auto
+    flex: 0
```

Comment:
```suggestion
```
by removing flex: 0 here we prevent the image from getting distorted on mobile devices (if the other fix above is applied too)

i could not see any problems with this change in the bigger resolution layouts,
if there are, this can be moved into the mobile media query too.

Reply:
Which device are you seeing this distortion? I'm using an LG G7 Thinq Android 8 with Chrome, as well as responsive tools on my desktop browser (admittedly, not the most reliable), and I don't see any issue.
Either way, your css suggestions don't do any harm from what I see, so I'll bring them in.